Assignment4/src/middleware/errorHandler.js

```diff
@@ -0,0 +1,7 @@
+const errorHandler = (err, req, res, next) => {
```

Comment:
Make this function more useful.Something went wrong will not help anyone to know or debug the issue.

Reply:
Done Sir...now i created error handling more dynamic using error.status and error.message properties